scripts/zkai_d64_rmsnorm_swiglu_statement_fixture.py

```
@@ -0,0 +1,479 @@
+            ],
+        },
+    }
+
+
+def validate_statement(statement: dict[str, Any]) -> None:
+    reference = evaluate_reference_block()
+    binding = commitments(reference)
+    expected = statement_payload(reference, binding)
+    for field, value in expected.items():
+        if statement.get(field) != value:
+            raise StatementFixtureError(f"statement field mismatch: {field}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)